src/download.c

```diff
@@ -301,6 +301,7 @@ int untar_full_download(void *data)
 	if (err) {
 		printf("ignoring tar extract failure for fullfile %s.tar (ret %d)\n",
 		       file->hash, err);
+		err = 0;
```

Comment:
I agree, we add to a failed list that is retried on multiple types of failures so err should be left alone. If we're still ignoring tar extract failures we should remove the printf, if that's something we don't want to ignore I'd change the printf to remove "ignoring" and make it a hard statement.